client/modules/IDE/components/Editor/MobileEditor.jsx

```diff
@@ -0,0 +1,90 @@
+import styled from 'styled-components';
+import { prop, remSize } from '../../../../theme';
+
+export const EditorContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  padding-bottom: 5rem;
+  transform: ${(props) =>
+    props.expanded ? 'translateX(50%)' : 'translateX(0)'};
+
+  > header {
+    display: flex;
+    ${prop('MobilePanel.secondary')}
```

Comment:
Looks like we’ve got a value with no property?  Here and on line 21.

Reply:
no this is to set the background color, it automatically does that

Reply:
we can also change all the `foreground` key in the theme to `color` to set the color and background automaticall by just writing this one line `${prop('some object background and color property')}`

Reply:
Right, I see what you’re saying.  You put the “color:” part in the theme object.  So it’s like a mixin rather than a color variable.